### DISCUSSION TITLE
latest compiled version of Fairy Stockfish with latest added variants

### DISCUSSION THREAD

**Discussion by musketeerchess** [fairy-stockfish/Fairy-Stockfish#743] [Q&A]
Hi, can someone post here the latest compiled version of stockfish where we have the latest developed chess variants (berolina chess etc). Can someone explain how to compile them?
Thanks

**Comment by ianfab** [comment]
You can always go to https://github.com/fairy-stockfish/Fairy-Stockfish/actions/workflows/release.yml?query=branch%3Amaster, click on the latest run, and download the zip there. That contains the latest development version. Unfortunately there is no permalink that always automatically points to the latest zip.
